patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback-contact-requested.twig

```diff
@@ -0,0 +1 @@
+{% include "@organisms/feedback/mass-feedback.twig" %}
```

Comment:
If this is just a variant, shouldn't it be created that way here?  with the ~variant on the json?
I think this file could be removed.